src/metpy/calc/tools.py

```diff
@@ -825,6 +825,47 @@ def lat_lon_grid_deltas(longitude, latitude, y_dim=-2, x_dim=-1, **kwargs):
     return dx * units.meter, dy * units.meter
 
 
+@exporter.export
+@preprocess_xarray
+def azimuth_range_to_lat_lon(azimuths, ranges, center_lon, center_lat, **kwargs):
+    """Convert azimuth and range bins in a polar coordinate system to lat/lon coordinates.
+
+    Pole refers to the origin of the coordinate system. Adapted from Brian Blaylock's code.
+
+    Parameters
+    ----------
+    azimuths :   array_like
+        array of azimuths defining the grid. If not a `pint.Quantity`,
+        assumed to be in degrees.
+    ranges  :    array_like
+        array of range distances from the pole. Typically in meters.
+    center_lat  :  float
+        The latitude of the pole in decimal degrees
+    center_lon :  float
+        The longitude of the pole in decimal degrees
+
+    Returns
+    -------
+    lat, lon:
+        two dimensional arrays of latitudes and longitudes corresponding
+        to the bin positions in the polar coordinate system
+
+    """
+    from pyproj import Geod
+
+    geod_args = {'ellps': 'sphere'}
+    if kwargs:
+        geod_args = kwargs
```

Comment:
This is the line codecov is dinging us on; in getting this covered could you expand and create a test for other ellipsoids?